### PR TITLE
Adds phpcs and resolves phpcs errors

### LINKED ISSUE #76: Add PHPCS to pipeline
Enforce CakePHP coding standards on the project with PHPCS

### PR DESCRIPTION
**Pull request by cnizzardini**
Resolves: #76

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
 
       - name: Test Suite
         run: |
-          composer test
+          composer check
```
Comment (dereuromark) [inline .github/workflows/pull-request.yml]: Wait, what is ci.yml then? If this is for CI?

Comment (it-thecmigroup) [reply .github/workflows/pull-request.yml:38]: I've traditionally split these up in my projects. But IDC how it gets done, as long as its done.

Comment (cnizzardini) [reply .github/workflows/pull-request.yml]: I've traditionally split these up in my projects, but IDC how it gets done just that it gets done.

Comment (dereuromark) [reply .github/workflows/pull-request.yml]: Would be good of those would be the same as for the majority of plugins
One file, and one badge in readme

Comment (cnizzardini) [reply .github/workflows/pull-request.yml]: Give that a look.

**Pull request merged**